lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpe.pm

```diff
@@ -0,0 +1,81 @@
+package Genome::Model::Tools::Bedpe::EvaluateBedpe;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Model::Tools::Bedpe::EvaluateBedpe {
+    is => 'Command::V2',
+    has_input => [
+        bedpe => {
+            is => 'File',
+            doc => 'bedpe file to be evaluated',
+        },
+        gold_bedpe => {
+            is => 'Path',
+            doc => 'bedpe file with gold standard breakpoints'
+        },
+        bedtools_version => {
+            is => 'Text',
+        },
+        slop => {
+            is => 'Integer',
+            doc => 'The amount of slop (in b.p.). to be added to each footprint of A. *Note*: Slop is subtracted from start1 and start2 and added to end1 and end2.',
```

Comment:
It's not really clear from the interface what `A` will be.  (I see in the code that it changes depending on the stat.)